examples/objects/src/main.pl

```diff
@@ -0,0 +1,40 @@
+use strict;
+use warnings;
+use 5.014;
+use FFI::Platypus 1.00;
+
+my $ffi = FFI::Platypus->new(
+  api  => 1,
+  lang => 'Rust',
+);
+$ffi->mangler(sub { "zip_code_database_$_[0]" });
+$ffi->find_lib( lib => 'objects' );
+$ffi->load_custom_type('::PtrObject' => 'ZipCodeDatabase' => 'ZipCodeDatabase' );
+
+package ZipCodeDatabase {
```

Comment:
Examples with a package inside of them use the package block syntax for readability.  This requires Perl 5.14, which was first released in 2011.  It might make sense for all of the examples to require the same version, but I've only specified versions where I need `say` and the package block syntax, and just the minimum perl required for each example.